__tests__/components/guided_experience_test.js

```diff
@@ -33,6 +33,7 @@ describe("GuidedExperience", () => {
       classes: {},
       id: "YY",
       nextSection: "ZZ",
+      benefitsDirectoryUrl: "/benefits-directory",
```

Comment:
At some point we should talk about making a Routes file just with the URLs as constants, so when the URL changes because of preference, we do not need to update the code. Somewhat similar to translation strings.

Reply:
👍 yeah, I was thinking something along those lines.